src/patternfly/demos/Page/page-template-header-tools-elements.hbs

```diff
@@ -1,15 +1,17 @@
 {{#> page-header-tools}}
   {{#> page-header-tools-group page-header-tools-group--modifier="pf-m-icons"}}
-    {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Alerts"'}}
+    {{#> button button--modifier="pf-m-plain pf-c-page__header-notifications" button--attribute='aria-label="Unread notifications"'}}
+      {{#> notification-badge notification-badge--modifier="pf-m-unread"}}
         <i class="fas fa-bell" aria-hidden="true"></i>
+      {{/notification-badge}}
     {{/button}}
-    {{#> button button--modifier="pf-m-plain" button--attribute='aria-label="Settings"'}}
+    {{#> button button--modifier="pf-m-plain pf-c-page__header-settings" button--attribute='aria-label="Settings"'}}
```

Comment:
Working with different apps, I think the settings button here is just to serve as an example of an icon that someone might put in their header. We're being very prescriptive with the things in the masthead currently, and I think we should be more generic, and only specific with things we know we want to control. So far I think we have 3 things we want to control:

* Notification badge - always shown
* User menu - shown on desktop
* Overflow menu - shown on mobile/tablet

Otherwise, I think we just need a class that allows for a button, dropdown, or whatever in it, and hides it on mobile/tablet. That may be a settings button/menu, help button, both, something else, etc...

I wonder if we should have like `pf-c-page__header-item` with modifiers for `pf-m-desktop` (shows on desktop, hides on mobile) and `pf-m-mobile` (shows on mobile, hides on desktop), so we would use no modifier for notification icon, the `pf-m-mobile` on the overflow menu, and `pf-m-desktop` on the settings/whatever icon and user menu?

We could also have specific classes like you have now for notification icon and user menu, then create a generic class like `pf-c-page__header-item` that you place things in that will show on desktop and hide on mobile, then if you want your menu to persist in both desktop/mobile, you just don't wrap the button in that `__header-item` element. 

I think the former might be easier to document and more intuitive. 

We could also extend the hidden/visible SCSS here and use that, though it's less prescriptive and would allow people to change the breakpoints at which things are hidden/shown more easily (which is good and bad).